src/Microsoft.ML.Transforms/Dracula/DraculaTransform.cs

```diff
@@ -0,0 +1,553 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
```

Comment:
Please rename the final to more relevant name. If we are calling it CountTragetEncoding then we should not be seeing Dracula in the codebase.